.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: ruby
 rvm:
-  - "2.3"
+  - "2.3.0"
   - "jruby-9.1.17.0"
```

Comment:
Can we also add jruby 1.7.23 to the support? 9.1 is the equivalent of ruby 2.3 and a lot of existing projects are using jruby 1.7.x.

It would also be good if everything passes to add 2.5 to the listed ones we test against as our newer projects are using 2.5.